### PR TITLE
[DEM-673] read the docs

### PR DESCRIPTION
**Pull request by rhjdjong**
To build the documentation, check out the code and install it with

```
pip install -e .[rtd,projectq,qiskit]
```

Then:

```
cd docs
make html
```

Then open `build/html/index.html` in your browser to see the documentation.

### CODE REVIEW

**Comment by IrinaVPushkina** [comment]
Ruud, the building of the docs is not that trivial. Can this be made more clear?

File: docs/examples.rst [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+Classifier
+----------
+
+.. toctree::
+
+..
+   examples/classifier_example/classification_example1_2_data_points
+   examples/classifier_example\classification_example2_4_data_points
+   examples/classifier_example\classification_example3_4_features
```
[the remark this answers is not in the document]
Comment (rhjdjong) [reply docs/examples.rst]: Repaired (and re-arranged the toctrees)

File: docs/examples.rst [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+   examples/example_qiskit_conditional
+
+Classifier
+----------
+
+.. toctree::
+
+..
+   examples/classifier_example/classification_example1_2_data_points
+   examples/classifier_example\classification_example2_4_data_points
```
[the remark this answers is not in the document]
Comment (rhjdjong) [reply docs/examples.rst]: Repaired (and re-arranged the toctrees)

File: docs/examples.rst [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+.. note::
+   The documentation for the examples must still be completed.
+
+ProjectQ
+--------
+.. toctree::
+
+..
+   examples/example_projectq
```
[the remark this answers is not in the document]
Comment (rhjdjong) [reply docs/examples.rst]: Added that one also.

File: docs/examples.rst [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+   examples/example_projectq_entangle
+
+Qiskit
+------
+.. toctree::
+
+..
+   examples/example_qiskit_conditional
```
[the remark this answers is not in the document]
Comment (rhjdjong) [reply docs/examples.rst]: Added that one also

File: src/quantuminspire/api.py [hunk 5]
```diff
@@ -57,26 +66,30 @@ def __init__(self, base_uri: str = QI_URL, authentication: Optional[coreapi.auth
...
-        Note: When no project name is given, a temporary project is created for the job and deleted after the job
-              has finished. When a project name is given, a project is created if it does not exist, but re-used
-              if a project with that name already exists. In either case, the project will not be deleted when a
-              project name is supplied here.
-
-        Raises:
-            AuthenticationError: An AuthenticationError exception is raised when no authentication is given
-                                 and the token could not be loaded from the default location.
-            ApiError: An ApiError exception is raised when the schema could not be loaded.
+        :param base_uri: The base uri of the Quantum Inspire API-location where the schema can be found
```
[the remark this answers is not in the document]
Comment (rhjdjong) [reply src/quantuminspire/api.py]: I copied the `conf` settings from qtt, and reverted to the Google Docs convention.

File: docs/examples.rst [hunk 6]
```diff
@@ -0,0 +1,39 @@
+Examples
+========
+
+.. note::
+   The documentation for the examples must still be completed.
```
[the remark this answers is not in the document]
Comment (rhjdjong) [reply docs/examples.rst]: This is meant as a reminder that the contents of (in particular) the classifier_example notebooks does not render correctly when building the documentation with Sphinx.

[the remark this answers is not in the document]
Comment (rhjdjong) [reply docs/examples.rst]: I moved this note to the section for the classifier examples

**Review by oschusler — APPROVED** [review]

**Review by QFer — APPROVED** [review]

**Pull request merged**